UPGRADE.md

```diff
@@ -2,6 +2,17 @@
 
 ## dev-develop
 
+### Media Format Cache Public Folder
+
+The `sulu_media.format_cache.public_folder` parameter does not longer exist, and therefore has to be removed from the configurationion. To set another public folder use the following configuration is needed.
```

Comment:
Maximum line length is 80 character, and you've got a typo in `configuration`. I would also stress more that you only have to change something if you've used the `sulu_media.format_cache.pblic_folder` parameter.